lib/accesslists/validate.go

```diff
@@ -52,19 +52,23 @@ func validateAccessList(a *accesslist.AccessList) error {
 		return trace.BadParameter("owners are missing")
 	}
 
-	if a.IsReviewable() {
+	if a.IsReviewable() || a.Spec.Audit.Recurrence.Frequency != 0 {
```

Comment:
Is the `a.IsReviewable()` condition now redundant both here and below?

Reply:
Or I guess we should still validate audit period if it's not set for a non-static list.